.golangci.yml

```diff
@@ -6,6 +6,9 @@ run:
   # worker.
   concurrency: 1
 
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  deadline: 5m
```

Comment:
What's the rationale for this change? Would be nice to include it in the commit message.

Reply:
just to try to make travis pass

Reply:
Removed